### PR TITLE
Added synchronization_databases_ag function.

### PR DESCRIPTION
**Pull request by matiasgrana**
Added synchronization_databases_ag function.

### CODE REVIEW

**Comment by matiasgrana** [comment]
Test:

nagios_sql -H xx.xx.xx.xx -U 'userxx ' -P 'Passwordxx' -t synchronization_databases_ag -S Server01
Ok: Name:Replication01 State:SYNCHRONIZING Health:HEALTHY
Name:Name:Replication02  State:SYNCHRONIZING Health:HEALTHY
Name:Name:Replication03  State:SYNCHRONIZING Health:HEALTHY

File: src/nagios_sql.py [hunk 1]
```diff
@@ -398,6 +399,55 @@ def availability_group_status(host, user, password):
 
     return {'code': code, 'msg': msg}
 
+@nagios_test
+def synchronization_databases_ag(host, user, password, server_primary):
+    """Databases Availability group Status"""
+	
+    msg = ''
+    primary = False
```
Comment (pablodav) [inline src/nagios_sql.py]: Remove the extra spaces after False here.
```suggestion
    primary = False
```

File: src/nagios_sql.py [hunk 2]
```diff
@@ -398,6 +399,55 @@ def availability_group_status(host, user, password):
...
+        
+        for row in rows:
+            
+            if row.get("synchronization_state_desc") != "SYNCHRONIZING":
+                synchronized = "CRITICAL"
+            
+            msg += "Name:{} State:{} Health:{}\n".format(
+                row.get("name"),
+                row.get("synchronization_state_desc"),
+                row.get("synchronization_health_desc"))
```
Comment (pablodav) [inline src/nagios_sql.py]: Remove the extra spaces
```suggestion
                row.get("synchronization_health_desc"))
```

File: src/nagios_sql.py [hunk 3]
```diff
@@ -412,10 +462,13 @@ def main():
     user = options.user
     password = options.password
     publisher_name = options.publisher_name
+    server_primary = options.server_primary
 
     func = get_func(test)
     if publisher_name:
         result = func(host, user=user, password=password, publisher_name=publisher_name)
+    elif server_primary:
+        result = func(host, user=user, password=password, server_primary=server_primary)
```
Comment (pablodav) [inline src/nagios_sql.py]: Remove the extra spaces
```suggestion
        result = func(host, user=user, password=password, server_primary=server_primary)
```

**Review by pablodav — COMMENTED** [review]
Thanks for adding the tests done!
Also for taking care of README.rst and all the good work done.

Just remove the extra spaces to allow app codacy pass:

https://app.codacy.com/app/pablodav/nagios_sql/pullRequest?prid=3216634

**Comment by pablodav** [comment]
Perfect I'm will merge later and create new version

**Pull request merged**